guides/plugins/plugins/framework/rule/add-custom-rules.md

```diff
@@ -199,3 +203,76 @@ The last step is, creating a template for our condition. We will create a new fi
 
 As you can see, our `sw-single-select` uses the previously created computed property `selectValues` as the `options` prop, and the value is saved into the variable `isLunarEclipse`. That's it, your rule is now fully integrated.
 
+## Context awareness
+
```

Comment:
Could you please add a hint here, mentioning from which Shopware Version this feature is supported?

Reply:
Done, that.